animalfacts.py

```diff
@@ -3155,6 +3174,7 @@ def animalfactsbot(reddit):
     WALLABY_FACTS,
     WALRUS_FACTS,
     WARTHOG_FACTS,
+    WILDEBEEST_FACTS,
```

Comment:
Really this should go after whale, please. The more we keep the list in order the easier it is to keep track of discrepancies.